config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql" if Rails.env.development?
```

Comment:
Nice, this will be helpful. I've been using [Insomnia](https://insomnia.rest/).

Reply:
Nice, I hadn't heard of Insomnia. I'm gonna check that out